osu.Game.Rulesets.Taiko/Difficulty/TaikoPerformanceCalculator.cs

```diff
@@ -43,6 +43,9 @@ protected override PerformanceAttributes CreatePerformanceAttributes(ScoreInfo s
             if (totalSuccessfulHits > 0)
                 effectiveMissCount = Math.Max(1.0, 1000.0 / totalSuccessfulHits) * countMiss;
 
+            // TODO: The detection of rulesets is temporary until the leftover old skills have been reworked.
+            bool taikoSpecificBeatmap = score.BeatmapInfo.Ruleset.OnlineID == 1;
```

Comment:
Would it be better to call this `isConvert` similar to everywhere else?

Reply:
That's a fair point.